components/CaseStatus/AddCaseStatusForm/AddCaseStatusForm.spec.tsx

```diff
@@ -65,3 +65,24 @@ describe('AddCaseStatusForm - CIN', () => {
     expect(getByText('this is a note')).toBeInTheDocument();
   });
 });
+
+describe('AddCaseStatusForm - CP', () => {
+  it('displays the form', () => {
+    const { getByText } = render(
+      <AddCaseStatusForm personId={mockedResident.id} prefilledFields={{}} />
+    );
+
+    expect(getByText('Child protection')).toBeInTheDocument();
+  });
+
```

Comment:
Should we test the validation?

Start date can be set in the past (at least on chrome)